packages/sit-onyx/src/components/OnyxSlider/OnyxSlider.vue

```diff
@@ -142,20 +151,24 @@ const isIconControl = computed(() => props.control === "icon" && props.mode ===
     >
       <template #default="{ id: inputId }">
         <div class="onyx-slider__container">
-          <div v-if="isValueControl" class="onyx-slider__control" aria-hidden="true">
-            {{ min }}
-          </div>
-
-          <div v-if="isIconControl" class="onyx-slider__control">
-            <OnyxIconButton
-              :disabled="disabled || (normalizedValues[0] ?? props.min) <= props.min"
-              :label="t('slider.decreaseValue', { n: shiftStep })"
-              color="neutral"
-              :icon="iconMinusSmall"
-              tabindex="0"
-              @click="handleDecreaseByIcon"
-            />
-          </div>
+          <OnyxSliderControl v-if="props.control === 'value'" control="value" :value="props.min" />
+          <OnyxSliderControl
+            v-if="props.control === 'icon' && props.mode === 'single'"
+            control="icon"
+            direction="decrease"
+            :shift-step="shiftStep"
+            :model-value="normalizedValues[0]"
+            :disabled="disabled || (normalizedValues[0] ?? props.min) <= props.min"
+            @update:model-value="handleSliderIconControlChange"
+          />
+          <OnyxSliderControl
+            v-if="props.control === 'input' && props.mode === 'range' && !props.discrete"
```

Comment:
For discrete mode it would still be great to input a value. If the value does not match the discrete steps, we can adjust it automatically :)